app/assets/javascripts/libs/error_handling.js

```diff
@@ -107,8 +107,22 @@ class ErrorHandling {
     };
   }
 
-  notify(error: Error, optParams: Object = {}) {
+  notify(error: Error, optParams: Object = {}, escalateToSlack: boolean = false) {
     this.airbrake.notify({ error, params: { actionLog: getActionLog(), ...optParams } });
```

Comment:
I'd extract the parameters object extension from here to the previous line, and then use the resulting object (with the attached action log) in the slack request as well :) Otherwise we won't see the action log in slack.

Reply:
Oh, very good point :+1: I completely missed that. I went for a bit different solution, so that the action log gets printed separately in the slack for the sake of readability :)